usr/lib/cca_stdll/cca_specific.c

```diff
@@ -239,6 +240,300 @@ static const CK_ULONG cca_mech_list_len =
                         (sizeof(cca_mech_list) / sizeof(MECH_LIST_ELEMENT));
 
 
+#ifdef DEBUG
+/* Debug function: hexdump buffer with TRACE_DEBUG */
+static void dump_mem(const char *prestr, void *buf, unsigned len)
```

Comment:
In ep11_specific.c there is a macro defined to dump memory:
`#define TRACE_DEBUG_DUMP(_buf, _buflen) hexdump(_buf, _buflen)`
Function hexdump() is alos local there. 
Maybe we can add a function into trace.c and a macro like TRACE_DEBUG_DUMP to use it, so that it can be used everywhere?

Reply:
yes, makes sense - will do this